packages/core/src/modules/ledger/LedgerModule.ts

```diff
@@ -19,6 +19,13 @@ export class LedgerModule {
     this.wallet = wallet
   }
 
+  /**
+   * Connect to all the ledger pools
+   */
+  public async connectToGenesis() {
+    await this.ledgerService.connectToPools()
+  }
```

Comment:
Maybe we should also call this `connectToPools`? That would be consistent with other method names and makes more sense than `connectToGenesis` to me.


```suggestion
  public async connectToPools() {
    await this.ledgerService.connectToPools()
  }
```

Reply:
+1